### DISCUSSION TITLE
Looking for maintainers

### DISCUSSION THREAD

**Discussion by danielmatthew** [talis/bootstrap-theme#34] [Q&A]
I'm leaving Talis on the 19th May, and I would like somebody to look after this codebase until my replacement arrives.

**Comment by danielmatthew** [comment]
@talis/team-aspire @talis/team-elevate 

Do you think you could take care of this? It's in use in Otis and Shipshape.

I've got a branch testing the theme against Bootstrap 5.3.0-alpha.2: the obvious gremlins have been resolved. It's been in alpha for a long, long time.

**Reply by junglebarry** [reply]
One for the entire tech team to consider, so I'll raise to Tech Leads.
